src/queries/vendors.js

```diff
@@ -0,0 +1,39 @@
+/**
+ * @name vendors
+ * @method
+ * @memberof Catalog/NoMeteorQueries
+ * @summary get an array containing all the values for the `product.vendor` field in the Catalog
+ * @param {Object} context - an object containing the per-request state
+ * @param {Object} params - request parameters
+ * @param {String[]} [params.shopIds] - Optional IDs of the shop to get a list of vendors for
+ * @param {String[]} [params.tagIds] - Optional IDs of the tags to get a list of vendors for
+ * @returns {Promise<MongoCursor>} - A MongoDB cursor for the proper query
```

Comment:
This seems to return an object not a `MongoCursor`

Reply:
Good catch!